packages/e2e-tests/specs/editor/plugins/iframed-inline-styles.test.js

```diff
@@ -1,56 +0,0 @@
-/**
- * WordPress dependencies
- */
-import {
-	activatePlugin,
-	createNewPost,
-	deactivatePlugin,
-	insertBlock,
-	getEditedPostContent,
-	canvas,
-	createNewTemplate,
-} from '@wordpress/e2e-test-utils';
-
-async function getComputedStyle( context, property ) {
-	await context.waitForSelector( '.wp-block-test-iframed-inline-styles' );
-	return await context.evaluate( ( prop ) => {
-		const container = document.querySelector(
-			'.wp-block-test-iframed-inline-styles'
-		);
-		return window.getComputedStyle( container )[ prop ];
-	}, property );
-}
-
-describe( 'iframed inline styles', () => {
-	beforeEach( async () => {
-		await activatePlugin( 'gutenberg-test-iframed-inline-styles' );
-		await createNewPost( { postType: 'page' } );
-	} );
-
-	afterEach( async () => {
-		await deactivatePlugin( 'gutenberg-test-iframed-inline-styles' );
-	} );
-
-	it( 'should load inline styles in iframe', async () => {
-		await insertBlock( 'Iframed Inline Styles' );
-
-		expect( await getEditedPostContent() ).toMatchSnapshot();
-		expect( await getComputedStyle( canvas(), 'padding' ) ).toBe( '20px' );
-		expect( await getComputedStyle( canvas(), 'border-width' ) ).toBe(
-			'2px'
-		);
-
-		await createNewTemplate( 'Iframed Test' );
```

Comment:
What did this do? Should we port it to Playwright too?

Reply:
This was used to switch to an iframed editor. The step is no longer needed as the iframed editor is the new default since WP 6.3.